homeassistant/components/webostv/media_player.py

```diff
@@ -326,7 +326,11 @@ def media_image_url(self) -> str | None:
     def supported_features(self) -> int:
         """Flag media player features that are supported."""
         if self.state == STATE_OFF and self._supported_features is not None:
-            return self._supported_features
+            return (
+                self._supported_features | SUPPORT_TURN_ON
+                if self._wrapper.turn_on
```

Comment:
We prefer not using a ternary operator expression if it spans more than one line.